### ISSUE TITLE
SystemPackageTool is broken on systems without sudo

### ISSUE THREAD

**Issue by Lyberta** [conan-io/conan#4770]
I don't have `sudo` on my systems because I consider it a gaping security hole. But `SystemPackageTool` still tries to invoke `sudo` like this:

```
Running: sudo apt-get update
sh: 1: sudo: not found
sdl2/2.0.9@bincrafters/stable: ERROR: while executing system_requirements(): Command 'sudo apt-get update' failed
ERROR: Error in system requirements
```

I think `SystemPackageTool` should check the existence of `sudo` before trying to call it.

**Comment by uilianries** [comment]
Hi @Lyberta 

You could set `export CONAN_SYSREQUIRES_SUDO=0`

https://docs.conan.io/en/latest/reference/env_vars.html#conan-sysrequires-sudo

**Comment by Lyberta** [comment]
I think regardless of the value of the variable, Conan should still check that sudo exists.

**Comment by uilianries** [comment]
Yes, I agree with you. This could be improved for the next release.

**Comment by memsharded** [comment]
Thanks @uilianries 

It has been merged, will be released in Conan 1.14
